### PR TITLE
feat(landing): flesh out content

### LINKED ISSUE #111: Landing page
[Figma reference for UI](https://www.figma.com/file/qtoA63mFtCW3jGYhr8ev7y/Master?node-id=638%3A3793)

​

Pending Kee Wei's review on Alwyn's PR

### PR DESCRIPTION
**Pull request by LoneRifle**
## Problem

Closes #111 

## Solution
- Add the rest of the content from wireframes
- Add a click handler on the Learn More button to scroll down

### CODE REVIEW

File: src/client/pages/Landing.tsx [hunk 1]
```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
...
             </Button>
           </VStack>
           <HStack h="100%" flex={1} justifyContent="center" />
-        </HStack>
+        </Stack>
       </Container>
+      <Container id="learn-more" maxW="100vw" pt="8vh" bgColor="neutral.50">
+        <Container
+          maxW="80vw"
+          alignItems="center"
```
Comment (lamkeewei) [inline src/client/pages/Landing.tsx]:
```suggestion
```
Think this is not required.

File: src/client/pages/Landing.tsx [hunk 2]
```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
...
+          justifyContent="space-between"
```
Comment (lamkeewei) [inline src/client/pages/Landing.tsx]:
```suggestion
```
Same for this.

File: src/client/pages/Landing.tsx [hunk 3]
```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
...
+              width="120px"
+            >
               Learn More
             </Button>
           </VStack>
           <HStack h="100%" flex={1} justifyContent="center" />
-        </HStack>
+        </Stack>
       </Container>
+      <Container id="learn-more" maxW="100vw" pt="8vh" bgColor="neutral.50">
```
Comment (lamkeewei) [inline src/client/pages/Landing.tsx]:
```suggestion
      <Container id="learn-more" maxW="100vw" py="8vh" bgColor="neutral.50">
```

Based on the design, we need padding at the bottom too. 

image

File: src/client/pages/Landing.tsx [hunk 4]
```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
...
+                checker through a shareable link.
+              </Text>
+            </VStack>
+          </Stack>
+        </Container>
+      </Container>
+      <Container maxW="100vw" py="8vh" bgColor="neutral.100">
+        <Container
+          maxW="80vw"
+          alignItems="center"
```
Comment (lamkeewei) [inline src/client/pages/Landing.tsx]:
```suggestion
```

File: src/client/pages/Landing.tsx [hunk 5]
```diff
@@ -126,17 +135,143 @@ export const Landing: FC = () => {
...
+          justifyContent="space-between"
```
Comment (lamkeewei) [inline src/client/pages/Landing.tsx]:
```suggestion
```

**Review by lamkeewei — APPROVED** [review]
Minor nits. Otherwise lgtm!

File: src/client/pages/Landing.tsx (hunk 5 above)
Comment (LoneRifle) [reply src/client/pages/Landing.tsx]: https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/ -_-

**Pull request merged**